src/main/java/org/minions/devfund/angela/moviedatabase/Movie.java

```diff
@@ -0,0 +1,78 @@
+package org.minions.devfund.angela.moviedatabase;
+
+import java.util.List;
+
+/**
+ * Manages the movie object.
+ */
+public class Movie {
+
+    private String name;
+    private List<Actor> actors;
+    private double rating;
+
+    /**
+     * Constructor of {@link Movie}.
+     *
+     * @param newName   String the movie name.
+     * @param newActors List with the actors in the movie.
+     */
+    public Movie(final String newName, final List<Actor> newActors) {
```

Comment:
Is it really needed to send newActors List ?
because the line of code that are using this 
is this one and you are sending null. :(
.orElse(new Movie("", null)).getName();

Consider removing it!

Reply:
Done